.github/workflows/qa.yml

```diff
@@ -0,0 +1,118 @@
+name: QA
+
+on:
+  workflow_dispatch:
+    inputs:
+      setup:
+        description: "Deployment setup:"
+        required: true
+        default: "north-america"
+        type: choice
+        options:
+          - global
+          - north-america
+      expr_duration:
+        description: "Experiment duration (in minutes)"
+        required: true
+        default: "15"
+        type: choice
+        options:
+          - "5"
+          - "10"
+          - "15"
+
+jobs:
+  run-malachite-do:
+    runs-on: ubuntu-latest
+    env:
+      TF_VAR_do_token: "${{ secrets.DO_TOKEN }}"
+      TF_VAR_ssh_keys: '["${{ secrets.DO_SSH_FINGERPRINT }}"]'
+      EXPR_SETUP: "${{ github.event.inputs.setup || 'north-america' }}"
+      EXPR_DURATION_MINUTES: "${{ github.event.inputs.expr_duration || '15' }}"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
Will need to add the branch to checkout here via `gitHub.event.inputs.branch` in order to satisfy https://github.com/informalsystems/malachite/pull/1023/files#r2088306372